_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,248 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+		- Brian Bors
+		- Wilco Fiers
+  previous_authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul`, `ol`, `dl`, and `menu` [HTML elements][] that is [included in the accessibility tree][] and has the same [semantic role][] as its [implicit semantic role][].
+
+## Expectation 1
+
+Each test target only has [child][] [node][] in the [flat tree][] for which one of the following is true:
+
+- the [child][] [element][] follows the [content model][]; or
+- the [child][] has an [explicit role][] which is the [Implicit ARIA semantics]() for an HTML element included in the [content model][] for the test target; or
+- the [child][] is not included in the [accessibility tree][].
+
+## Expectation 2
+
+Each test target has one or more [child][] [elements][] in the [flat tree][] that is [included in the accessibility tree][].
+
+## Assumptions
+
+_There are currently no assumptions._
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [HTML Specification - Content model](https://html.spec.whatwg.org/#concept-element-content-model)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This [HTML `<ul>` element][] has a [descendant][] of type [HTML `<li>` element][].
+
+```html
+<ul>
+	<li>Eggs</li>
+	<li>Milk</li>
+</ul>
+```
+
+#### Passed Example 2
+
+This [HTML `<dl>` element][] contains at least one name-value group as [descendant][].
+
+```html
+<dl>
+	<dt>Physics</dt>
+	<dd>8/10</dd>
+</dl>
+```
+
+#### Passed Example 3
+
+This [HTML `<menu>` element][] has a [descendant][] of type [HTML `<li>` element][].
+
+```html
+<menu>
+	<li></li>
+</menu>
+```
+
+#### Passed Example 4
+
+This [HTML `<ol>` element][] contains at least one [descendant][] of type [HTML `<li>` element][].
```

Comment:
This description is not correct. Should be in the line of "with explicit role equal to the implicit role of `li`" or something like that.